src/main/java/bio/terra/common/db/DatabaseProperties.java

```diff
@@ -0,0 +1,116 @@
+package bio.terra.common.db;
+
+import static org.apache.commons.pool2.impl.GenericObjectPoolConfig.DEFAULT_MAX_TOTAL;
+
+import org.apache.commons.dbcp2.*;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/** Defines configuration properties for using {@link DatabaseConfiguration} use. */
+public class DatabaseProperties {
```

Comment:
define this as abstract if you want it to be used by subclasses only. Maybe name it `BaseDatabaseProperties`.

Reply:
This can be used as a non-abstract class in a non-DI world. But there is no Terra service use it in that way. 
Not sure if we want to make it abstract and subclass-only or keep it in that way. 
@ddietterich wdyt? you implement the 1st version then we keep using this pattern since then.

Reply:
updated, add a abstract base and a PrimaryDbConfig. Does that LG?

Reply:
change my mind after sync after standup.